pubspec.yaml

```diff
@@ -5,7 +5,7 @@ author: zmtzawqlp <zmtzawqlp@live.com>
 homepage: https://github.com/fluttercandies/like_button
 
 environment:
-  sdk: ">=2.1.0 <3.0.0"
+  sdk: ">=2.3.0 <3.0.0"
```

Comment:
In what reason this should be higher than 2.3.0?

Reply:
Yes

Reply:
Why needs higher version?